pkg/placement/controller.go

```diff
@@ -75,17 +75,20 @@ var excludedResourceNames = map[string]bool{
 type Controller struct {
 	ctx              context.Context
 	logger           logr.Logger
-	ocmClient        client.Client
+	ocmClient        client.Client // TODO (maroon): this should be deleted when transport is ready
 	dynamicClient    *dynamic.DynamicClient
 	kubernetesClient *kubernetes.Clientset
 	extClient        *apiextensionsclientset.Clientset
```

Comment:
didn't check but I assume there is Interface as well..
that was actually important comment I got from Mike in my PR.
after code is ready and manually tested we will probably want to add unit tests